Kudu.Core/Jobs/JobLogger.cs

```diff
@@ -68,6 +68,20 @@ protected string GetStatusFilePath()
         public void ReportStatus<TJobStatus>(TJobStatus status) where TJobStatus : class, IJobStatus
         {
             ReportStatus(status, logStatus: true);
+            if (status.GetType() == typeof(ContinuousJobStatus))
```

Comment:
yes pushed, and tests passed. Suwat took a look and had no comments. yes it makes more sense to have that in a method for each, will do that quickly